tests/files/tests.py

```diff
@@ -200,6 +200,23 @@ def test_shared_lock(self):
             self.assertIs(locks.unlock(f1), True)
             self.assertIs(locks.unlock(f2), True)
 
+    def test_open_supports_full_signature(self):
+        """
+        File.open supports full signature of Python's open.
+        """
```

Comment:
Are we still asking folks to remove these? 🤔

Reply:
Yes, we can remove it. It's the same as the method name.